torch/lib/c10d/ProcessGroup.cpp

```diff
@@ -55,8 +55,11 @@ bool ProcessGroup::Work::wait(std::chrono::milliseconds timeout) {
   return true;
 }
 
-void ProcessGroup::Work::abort() {
-  TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.")
+void ProcessGroup::Work::abort(){
+    TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.")}
```

Comment:
Looks like the formatting messed up here.

Reply:
VS Code fixed that automatically. If I do 
```
void ProcessGroup::Work::abort(){
    TORCH_CHECK(false, "ProcessGroup::Work::abort not implemented.")
}
```
It gives a clang format warning.